tox.ini

```diff
@@ -12,15 +12,13 @@ deps =
     pytest
     pytest-cov
     dnspython
-    jsonschema
```

Comment:
IIUC we will reintroduce these changes(and the next one) once the required dependencies are sorted out within distribution, right?